src/components/ShopDropdownMenu/index.tsx

```diff
@@ -1,20 +1,20 @@
 import { ShopDropdownMenuProps } from '@/types';
 import s from './ShopDropdownMenu.module.scss';
 import { storePanelButtonsList } from '@/constants/storePanelButtonsList';
-import { NavLink } from 'react-router-dom';
+import { Link, NavLink } from 'react-router-dom';
+import { Icon } from '@iconify/react';
 
 export const ShopDropdownMenu = ({ setDropdownOpen }: ShopDropdownMenuProps) => {
 
   
   return (
-    <div className={s.shopDropdown}>
+    <div className={s.shopDropdown} onClick={() => setDropdownOpen(false)}>
       {true ? (
         <div>
           {storePanelButtonsList.map((button) => (
             <NavLink
               to={button.pathToPage}
```

Comment:
Links are incorrect and lead to a 404 page

Reply:
im not sure if we have correct ones because the shop page is in development, i just kept the ones we had in project within storePanelButtonsList

Reply:
If you change it to this, everything will work fine 
to={`/account/store/${item.pathToPage}`}